src/tool/python/settings.h

```diff
@@ -6,7 +6,7 @@ namespace xlang
     {
         std::set<std::string> input;
 
-        std::string output_folder;
+        std::experimental::filesystem::path output_folder;
```

Comment:
>std::experimental::filesystem [](start = 8, length = 29)

how come you're not using <filesystem> ?

Reply:
<filesystem> (wow, CodeFlow just snipped that out)

Reply:
I think there was a clang/ubuntu issue. Once we get Ubuntu build up and running, we should change this. Filed #85 to track